### PR TITLE
feat: add status icons to the delegate monitor

### LINKED ISSUE #664: feat: forging status icons
Add forging status icons to the delegate monitor instead of relying on only a colored circle. This is currently with Oleg to design fitting icons.

### PR DESCRIPTION
**Pull request by ItsANameToo**
Adds status icons to the delegate monitor, resolves #664 

image

## What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactor
- [ ] Performance
- [ ] Tests
- [ ] Build
- [ ] Documentation
- [ ] Code style update
- [ ] Continuous Integration
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch
- [x] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

**Comment by dated** [comment]
Nice! Minor nitpick: i'd swap the shapes in the `missing` icon, since it'd better represent `forged` -> `missed`

**Comment by ItsANameToo** [comment]
How so? right now you have `o o` for `forged forged`, then `- o` for `missed forged` and `- -` for `missed missed` (so latest block indicator is in front) 🤔

**Comment by dated** [comment]
Following the directionality (left to right) of the writing system the (presumabily) vast majority of current visitors uses it should be `o -` / `forged missed` :D

**Comment by ItsANameToo** [comment]
Ok misunderstanding from my  side, it's just an icon to show a missed block and not necessarily reflect the actual state of the last two blocks. E.g. if you would miss one, then forge, you'll be "forging`o o` " again even tho you had missed a block before which would technically mean `- o`

**Comment by dated** [comment]
As i said, minor nitpick. I can live with it this way :trollface:

**Review by faustbrian — APPROVED** [review]

File: resources/views/components/search/block.blade.php [hunk 1]
```diff
@@ -132,7 +118,7 @@ class="w-full dark:text-theme-secondary-200 dark:bg-theme-secondary-900"
     </div>
 </x-general.search.advanced-option>
 
-<x-general.search.advanced-option :title="trans('forms.search.date_range')">
+<x-general.search.advanced-option :title="trans('forms.search.date_range')" type="block">
```
Comment (ItsANameToo) [inline resources/views/components/search/block.blade.php:121]: `>` tsk

**Comment by alfonsobries** [comment]
@ItsANameToo sorry about that, not sure how can I miss that, I'm going to pay more attention in the future

It should be working now

**Pull request merged**